Automattic/woocommerce-payments
# Prevent Stripe Link from triggering the checkout fields warning

#### Changes proposed in this Pull Request

In #8339 we added a check for checkout fields modification and a warning explaining this might cause conflicts with Express Checkout methods.
When Stripe Link is enabled it changes the priority of the email field to display it first. In doing so, it makes it look like a new field is being added.
This PR checks if the email field exists before setting the priority to avoid a false positive.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/090b51cc-d539-4786-8782-e6e4a3105c3b)


#### Testing instructions

1. Using `develop`, go to the settings page and ensure Stripe Link is disabled and there's no warning.
2. Enable Stripe Link, refresh the page, and notice the warning is displayed.
3. Switch to `fix-stripe-link-checkout-warning` and ensure the warning is not displayed.

-------------------


**Post merge**
